apidoc/released/icu4c/appendable_8h_source.html

```diff
@@ -18,8 +18,8 @@
  <tbody>
  <tr style="height: 56px;">
   <td id="projectalign" style="padding-left: 0.5em;">
-   <div id="projectname">ICU 66.1
```

Comment:
It's too bad that this is in every single file. If only HTML had a good way to #include another file, then it wouldn't need to rev every file...